### PR TITLE
feat: update to more modern version of protogen and grpc-js

### PR DESCRIPTION
**Pull request by cprice404**
This commit updates us to the latest versions of the generated
client-protos, which includes a newer version of protogen and
grpc-js.  This was inspired by this comment related to a bug
that we recently discovered:

https://github.com/momentohq/client-sdk-nodejs/pull/488#issuecomment-1534010405

Haven't run the new test case yet to confirm whether this addresses
it, but upgrading to the latest should be for the best regardless.

### CODE REVIEW

**Review by bruuuuuuuce — APPROVED** [review]
:shipit:

File: packages/client-sdk-nodejs/src/internal/data-client.ts [hunk 1]
```diff
@@ -2213,6 +2214,12 @@ export class DataClient {
     sortedSetName: Uint8Array,
     value: Uint8Array
   ): Promise<CacheSortedSetGetRank.Response> {
+    console.log(`
```
Comment (krispraws) [inline packages/client-sdk-nodejs/src/internal/data-client.ts:2217]: Nit: Do we want this log message in the public sdk?

**Review by krispraws — APPROVED** [review]
Nitpick comment on the log message.

File: packages/client-sdk-nodejs/src/internal/data-client.ts [hunk 2]
```diff
@@ -2226,9 +2233,12 @@ export class DataClient {
...
           (err, resp) => {
-            if (resp?.missing) {
+            if (
+              resp?.missing ||
+              resp?.element_rank?.result === ECacheResult.Miss
+            ) {
               resolve(new CacheSortedSetGetRank.Miss());
-            } else if (resp?.element_rank) {
+            } else if (resp?.element_rank?.result === ECacheResult.Hit) {
               if (resp?.element_rank.rank === undefined) {
```
Comment (krispraws) [inline packages/client-sdk-nodejs/src/internal/data-client.ts:2242]: Late thought - `if (resp?.element_rank.rank === undefined)` can never be true with the new protogen ts files if it is a hit, do we still need this check?

File: packages/client-sdk-nodejs/src/internal/data-client.ts [hunk 3]
```diff
@@ -2213,6 +2214,12 @@ export class DataClient {
     sortedSetName: Uint8Array,
     value: Uint8Array
   ): Promise<CacheSortedSetGetRank.Response> {
+    console.log(`
+
+    SEND SORTED SET GET RANK YO
+
+    `);
+
```
Comment (cprice404) [inline packages/client-sdk-nodejs/src/internal/data-client.ts:2222]:
```suggestion
```

**Review by krispraws — APPROVED** [review]
:shipit:

**Pull request merged**